lib/tasks/incoming_messages.rake

```diff
@@ -9,4 +9,9 @@ namespace :incoming_messages do
       puts "Parsed #{msg.id} in #{ bm.real.round(6) }" if verbose
     end
   end
+
+  desc 'Update InfoRequest#incoming_messages_count counter cache'
+  task update_counter_cache: :environment do
+    InfoRequest.update_all('incoming_messages_count = (SELECT COUNT(*) FROM "incoming_messages" WHERE "incoming_messages"."info_request_id" = "info_requests"."id")')
```

Comment:
Any way to estimate how long this will take to run in prod?

Reply:
Not very long: https://gist.github.com/gbp/538369487c20a1369aeb42c2622cd987